tests/test_preprocess_artifacts.py

```diff
@@ -1,71 +1,74 @@
 import inspect
 import os
+import pathlib
+
 import pytest
 
 from .utils import LocalTestEnvironment
+from .conftest import FuzzyCallChecker
+
+
+class ArtifactsTestEnvironment(LocalTestEnvironment):
```

Comment:
The logic was not changed. 
Code was moved to declare the `ArtifactsTestEnvironment` before usage.
Typing was added here too.